dashboard/callbacks/tmobile.py

```diff
@@ -105,6 +106,30 @@ def update_graphs_using_status_clicks(click_filter, project_name):
     return {'data': None, 'layout': None}, {'data': None, 'layout': None}
 
 
+@app.callback(
+    [
+        Output('redenna_project_t-mobile', 'figure')
+    ],
+    [
+        Input('status-count-filter-t-mobile', 'data'),
+        Input('project-dropdown-tmobile', 'value')
```

Comment:
Als dit niet perse een trigger voor de callback is maar slechts benodigde info, zou je hier ook een State van kunnen maken (r115).

Reply:
Als je in de dropdown een ander project kiest moet de grafiek ook geupdate worden